gradle.properties

```diff
@@ -8,3 +8,5 @@ ossrhUsername=your-jira-id
 ossrhPassword=your-jira-password
 
 selenium.version=4.5.0
+# Please increment the value in a release
```

Comment:
@SrinivasanTarget Do you usually bump this version manually upon releasing the package or there is some script for that? Shall we sync more java client release scripts/tutorials about this change?

Reply:
would you have some time to document all steps of the release procedure?

Reply:
Yes I will do it in the following days.